config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+6UtdL20nDcU/h1e/s+cLuMV8OKULpWdob9Q8T9seSN5CkUGXv/xFIh46S/NSF2b+yQ/r2jhDpp8pJv0TKcm7l75CMmpGnWdlN4Hrb/9n8dZFXSuOtkCyOqzInH1QxolBRxllD3Z4Dntmd6fLd9q2ObKYklK1uPHQWLj7x4cydkOq7o560GNhfx/FrRXH7fzcqijyeO0C8VFPWwfoS9ixVaxAt7xIx11B86+RvJIiYY8tD6IxxOucQAH8gjpfQOnnXI9EhGX3b9pQWr7fSAbfzAc1scoRxvm/7NNXvY6OFNlsrHKNBHrkoi5oRlq1xGpC2hnfRDjSBVaCXHsjR8SUy3/G7sPlPGI9alMakKkG3HYiPmJ9HH+P7hqpw7JnCMIKQh2dp+4T9FDyQd/3XClzmeDih3emK0uDIs+x--D9NGUrjckhCra6rx--3o7vuYqG5aXSFwk3xuE5fA==
```

Comment:
create `credentials.yml.enc.example` and push to git, this file you add to gitignore

Reply:
done